accord-core/src/main/java/accord/local/NodeTimeService.java

```diff
@@ -36,18 +37,53 @@ public interface NodeTimeService
     long now();
 
     /**
-     * Return the current time since the Unix epoch in the specified time unit. May still be simulated time and not
-     * real time.
+     * Return the current time since some arbitrary epoch in the specified time unit. May still be simulated time and not
+     * real time. The time returned by this will be monotonic.
      */
-    long unix(TimeUnit unit);
+    long elapsed(TimeUnit unit);
```

Comment:
I don't think this change is good for the project.  We do require `unix` and not `elapsed` in most places, and `elapsed` is relative to the local node only and not cross the cluster.  This logic is used as part of our `TxnId` so switching to `elapsed` *should* break us in subtle ways.